jdaviz/app.py

```diff
@@ -295,14 +315,20 @@ def _on_snackbar_message(self, msg):
         # * info lets everything through
         # * success, secondary, and primary are treated as info (not sure what they are used for)
         # * None is also treated as info (when color is not set)
-        ignored = []
-        if self.verbosity in ('warning', 'error'):
-            ignored += ['success', 'info', 'secondary', 'primary', None]
-            if self.verbosity == 'error':
-                ignored.append('warning')
-
-        if msg.color not in ignored:
-            self.state.snackbar_queue.put(self.state, msg)
+        levels = ['debug', 'info', 'warning', 'error']
```

Comment:
Should `levels` be made a class attribute, so we can reuse it in the setters?

Reply:
as something like `app._verbosity_levels` or even just a `_verbosity_levels` at the top-level in `app.py`?  I don't think its something we want to muddy up public attributes, but I like the idea of putting it somewhere accessible by the setters to avoid it being defined in multiple places

Reply:
Hidden attribute is fine by me, as this is for internal use only. :)